### PR TITLE
Stats: Create Measure from StatsRecorder.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/bogdandrutu/openconsensus/issues/185.

Second part of https://github.com/bogdandrutu/openconsensus/issues/185.

### CODE REVIEW

File: api/src/main/java/openconsensus/stats/Measure.java [hunk 1]
```diff
@@ -92,18 +92,15 @@ private Measure() {}
   @AutoValue
   public abstract static class MeasureDouble extends Measure {
 
-    MeasureDouble() {}
```
Comment (songy23) [inline api/src/main/java/openconsensus/stats/Measure.java]: This allows `MeasureDouble` to be extended in implementation.

File: api/src/main/java/openconsensus/stats/Measure.java [hunk 2]
```diff
@@ -92,18 +92,15 @@ private Measure() {}
...
      * Constructs a new {@link MeasureDouble}.
      *
      * @param name name of {@code Measure}. Suggested format: {@code <web_host>/<path>}.
      * @param description description of {@code Measure}.
      * @param unit unit of {@code Measure}.
      * @return a {@code MeasureDouble}.
-     * @since 0.1.0
      */
-    public static MeasureDouble create(String name, String description, String unit) {
+    static MeasureDouble create(String name, String description, String unit) {
```
Comment (songy23) [inline api/src/main/java/openconsensus/stats/Measure.java]: This method is no longer exposed and only used in Noop.

**Comment by bogdandrutu** [comment]
Working on this on a different PR, I will post it in few minutes, if you can take a look soon.

**Comment by songy23** [comment]
SGTM

**Pull request closed without merging**